src/js/button.js

```diff
@@ -7,13 +12,13 @@
  * @class
  * @constructor
  */
-vjs.Button = vjs.Component.extend({
+var Button = Component.extend({
   /**
    * @constructor
    * @inheritDoc
    */
   init: function(player, options){
```

Comment:
First off, a migration guide would be incredibly valuable and probably way more useful coming from someone that's actually doing it, so... :100::thumbsup:

So the goal is that this shouldn't change *that* much on the outside. There will need to be some changes regarding how one extends a VJS class, but beyond that things should be fairly straight forward (famous last words).